helm-chart/README.md

```diff
@@ -2,6 +2,8 @@
 
 If you'd like to deploy Online Boutique via its Helm chart, you could leverage the following instructions.
 
+**Warning:** Online Boutique's Helm chart is currently experimental. If you have feedback or run into issues, let us know inside [GitHub Issue #1319](https://github.com/GoogleCloudPlatform/microservices-demo/issues/1319) or by creating a [new GitHub Issue](https://github.com/GoogleCloudPlatform/microservices-demo/issues/new/choose).
```

Comment:
**My rationale for this change:**
I've added a warning about these Helm charts being in experimental mode. We can remove this note once:
1. We have at least 1 review from another Helm expert from our team.
2. We have figured out how Helm fits into our current release process.
3. We are confident with the amount of documentation around these Helm charts (for both Online Boutique maintainers and users).